### DISCUSSION TITLE
How to make Ctrl+R go back through history (like in bash by default), rather than "toggle relevance"?

### DISCUSSION THREAD

**Discussion by shamefulCake1** [junegunn/fzf#3816] [Q&A]
I am used to typing Ctrl+R, typing in some search keywords, and then using Ctrl+R/Ctrl+S to scroll through history, like default bash does.

Can this be done with fzf?

**Comment by LangLangBart** [comment]
Try to assign `FZF_CTRL_R_OPTS`[^1], e.g.:

```bash
export FZF_CTRL_R_OPTS="
  --bind 'ctrl-r:up'
  --bind 'ctrl-s:down'"
```
[^1]: [README.md - Key bindings for command-line](https://github.com/junegunn/fzf#key-bindings-for-command-line)

**Comment by shamefulCake1** [comment]
Perfect!
